packages/cli/src/daemon/doc/IAnswerQuestions.ts

```diff
@@ -0,0 +1,22 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+/**
+ * Specifies wwhether a question can be asked of the user, and if not,
+ * what value should be used by default.
+ */
+export interface IAnswerQuestions {
```

Comment:
I think the name `IAnswerQuestions` seems rather generic for an interface that is used specifically for daemon mode. Would it make sense to name this interface something more specific like `IDaemonEnablePrompt`? On the other hand, maybe the current name is fine since this interface isn't exported in the top level `index.ts`.

Reply:
I agree with the name change. While in there, I will also simplify the structure. We only need to know if we are allowed to ask the user any questions. We do not need a 'can I ask' property for each possible question.